app.rb

```diff
@@ -0,0 +1,80 @@
+require_relative './lib/email_notice.rb'
+require_relative './lib/input_checkers.rb'
+require_relative './lib/peep_manager.rb'
+require_relative './lib/users.rb'
+require 'sinatra/base'
+require 'sinatra/flash'
+
+class Chitter < Sinatra::Base
+
+  enable :sessions
+  register Sinatra::Flash
+
+  get '/' do
+    @current_user = session[:current_user]
+    erb :welcome
+  end
+
+  get '/peep_feed' do
+    @current_user = session[:current_user]
+    @peeps = Peeps.all
+    erb :peep_feed
+  end
+
+  get '/post_peep' do
+    erb :post_peep
+  end
+
+  post '/post_peep' do
```

Comment:
Why is your route called 'post_peep'? Surely it should be just be post 'peep'? You already have post in the verb, it doesn't need to be in the route.